pelican-bootstrap3/templates/includes/comments.html

```diff
@@ -8,7 +8,7 @@
             /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
             var disqus_shortname = '{{ DISQUS_SITENAME }}'; // required: replace example with your forum shortname
 
-            {% if article %}
+            /*{% if article %}
```

Comment:
Why comment this out? Is there a reason to comment this out instead of removing it?